pom.xml

```diff
@@ -55,6 +55,9 @@
         <module>modules/camel-ext</module>
         <module>modules/jms11-ext</module>
         <module>modules/kafka-ext</module>
+        <module>modules/spring-data</module>
+        <module>modules/spring-data-2.0</module>
+        <module>modules/spring-data-2.2</module>
```

Comment:
We can rename these modules to `sprint-data-ext`, `spring-data-2.0-ext`and `spring-data-2.2-ext`